piet-test/src/picture_5.rs

```diff
@@ -0,0 +1,25 @@
+//! Basic example of just text
+
+use piet::kurbo::Line;
+use piet::{Color, Error, FontBuilder, RenderContext, Text, TextLayout, TextLayoutBuilder};
+
+pub fn draw(rc: &mut impl RenderContext) -> Result<(), Error> {
+    // Black background
+    rc.clear(Color::BLACK);
+
+    // do something texty
+    let font = rc.text().new_font_by_name("Segoe UI", 12.0)?.build()?;
+
+    let layout = rc.text().new_text_layout(&font, "piet text!")?.build()?;
+
+    let width: f64 = layout.width().into();
```

Comment:
The `into` is not needed here, nor is the type annotation, as we got rid of the associated type (which was designed to be either f32 or f64).

Reply:
I cleaned this up and amended the commit.